portfolio/index.html

```diff
@@ -36,6 +36,7 @@ <h3>About Me</h3>
             </p>
         </section>
         <section class="codingExp">
+            <!--Coding Experience Section-->
```

Comment:
smart idea to comment the section title to make it more readable and easy to navigate